test/core/test_serde.py

```diff
@@ -21,6 +21,7 @@
 from pydantic import BaseModel
 
 from gluonts.core import serde
+from gluonts.mx.prelude import *
```

Comment:
Not sure why this import is needed: I’m guessing to bring names into scope so that deserialization works, but I don’t see it used in the test changed by this PR

Reply:
I couldn't run the test by itself, since it's relying on being able to serialize numpy and mxnet arrays.